package.json

```diff
@@ -63,8 +63,12 @@
     "assert-rejects": "^1.0.0",
     "c8": "^7.1.0",
     "codecov": "^3.6.5",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.0.0",
+    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-prettier": "^3.1.0",
     "espower-typescript": "^9.0.2",
-    "gts": "^1.1.2",
+    "gts": "^2.0.0-alpha.4",
```

Comment:
why not `@next`?

Reply:
Can you please try with the next? I have a feeling the formatting might change with the version change.

Reply:
oh, i ran `npm install gts@next` then it's updated like this, should I manually change it to `"gts": "@next"`? Thanks

Reply:
updated it to `"gts": "next"`

Reply:
Forget it, it's THE SAME version.